src/modules/Security.rb

```diff
@@ -597,12 +520,136 @@ def WriteConsoleShutdown(ca)
       true
     end
 
+    # Write the settings from @Locations to the corresponding files
+    def write_to_locations
+      commitlist = []
+      # NOTE: the call to #sort is only needed to satisfy the old testsuite
+      @Locations.sort.each do |file, vars|
+        vars.each do |var|
+          val = @Settings[var]
+          if val && val != SCR.Read(path("#{file}.#{var}"))
+            SCR.Write(path("#{file}.#{var}"), val)
```

Comment:
Well, this is only backporting. It's the same code in the master branch, so I'd rather leave it as it is. I only want to introduce changes that are needed for the whole thing to run in SLE12-GA.